### ISSUE TITLE
prov/verbs fails with the fi_getinfo_test

### ISSUE THREAD

**Issue by gladkovdmitry17** [ofiwg/libfabric#3488]
Since new the `fi_getinfo_test` has been extended in the https://github.com/ofiwg/fabtests/pull/748, the Verbs provider should be fixed to be compliant with those changes

```
$ ./fi_getinfo_test -p verbs
Running getinfo_no_hints1 [Test with no node, service, flags or hints]...PASS!
Running getinfo_no_hints2 [Test with node, no service, FI_SOURCE flag and no hints]...PASS!
Running getinfo_no_hints3 [Test with service, FI_SOURCE flag and no node or hints]...PASS!
Running getinfo_no_hints4 [Test with node, service, FI_SOURCE flags and no hints]...PASS!
Running getinfo_no_hints5 [Test with node, service and no hints]...PASS!
Running getinfo_src1 [Test with no node, service, or flags]...PASS!
Running getinfo_src2 [Test with node, no service, FI_SOURCE flag]...PASS!
Running getinfo_src3 [Test with service, FI_SOURCE flag and no node]...PASS!
Running getinfo_src4 [Test with node, service, FI_SOURCE flags]...PASS!
Running getinfo_src_dest1 [Test with node, service]...PASS!
Running getinfo_src_dest2 [Test API version]...PASS!
Running getinfo_msg_ordering1 [Test msg ordering bits supported are set]...PASS!
Running getinfo_raw_ordering1 [Test rma RAW ordering size is set]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_raw_ordering2 [Test rma RAW ordering size is set to hints]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_war_ordering1 [Test rma WAR ordering size is set]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_war_ordering2 [Test rma WAR ordering size is set to hints]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_waw_ordering1 [Test rma WAW ordering size is set]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_waw_ordering2 [Test rma WAW ordering size is set to hints]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_bad_raw_ordering1 [Test invalid rma RAW ordering size]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_bad_war_ordering1 [Test invalid rma WAR ordering size]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_bad_waw_ordering1 [Test invalid rma WAW ordering size]...skipped because: fi_getinfo failed No data available(61)
Running getinfo_neg1 [Test with non-existent domain name]...PASS!
Running getinfo_mr_mode1 [Test FI_MR_BASIC]...FAIL: fi_getinfo failed No data available(61)
Running getinfo_mr_mode2 [Test FI_MR_SCALABLE]...PASS!
Running getinfo_mr_mode3 [Test FI_MR_UNSPEC (v1.0)]...PASS!
Running getinfo_mr_mode4 [Test FI_MR_BASIC (v1.0)]...PASS!
Running getinfo_mr_mode5 [Test FI_MR_SCALABLE (v1.0)]...PASS!
Running getinfo_mr_mode6 [Test mr_mode bits]...FAIL: fi_getinfo failed Unspecified error(256)

Summary: 2 tests failed
```

**Comment by a-ilango** [comment]
This should hopefully get fixed with #3469

**Comment by gladkovdmitry17** [comment]
> This should hopefully get fixed with #3469

Thanks. I'll check
Because we use utility checkers, it will be get fixed w/o any changes in the verbs provider. Nice :)

**Comment by gladkovdmitry17** [comment]
@shefty @a-ilango Thanks. It really fixes the verbs provider. Nice to have utility code instead of self-implemented. It allows us to get the problem fixed w/o any changes in the provider code :)
